addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -341,7 +327,7 @@
          */
         _onClickTab: function (ev) {
             ev.stopPropagation();
-            var $elem = $(ev.currentTarget);
+            const $elem = $(ev.currentTarget).parents('.o_wslides_fs_sidebar_list_item');
```

Comment:
Are we really looking for multiple parents here?
If not then maybe "closest" is more appropriated.